src/main/java/com/epam/tc/hw/mobile/setup/BaseTest.java

```diff
@@ -12,11 +12,11 @@
 
 public class BaseTest implements IDriver {
 
-    private static AppiumDriver appiumDriver; // singleton
+    private static AppiumDriver<?> appiumDriver; // singleton
     IPageObject po;
```

Comment:
The variable is never used in the tests, please, remove it or use somewhere

Reply:
removed